includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -101,7 +101,22 @@ function perflab_aao_autoloaded_options_test() {
  */
 function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
-	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
+
+	if ( function_exists( 'wp_autoload_values_to_autoload' ) ) {
+		$autoload_values = wp_autoload_values_to_autoload();
+	} else {
+		$autoload_values = array( 'yes' );
+	}
```

Comment:
Rather than repeating this several times, how about consolidating it to our own `perflab_aao_get_autoload_values()` helper?

Reply:
@joemcgill Great idea, done in https://github.com/WordPress/performance/pull/1112/commits/031658609d93296d6eb556f086a78eba92d879e2